.gitignore

```diff
@@ -9,4 +9,4 @@ dist/
 *~
 /tests/fixtures/date_adjustment/testing_output.csv
 /tests/testing_output.csv
-/.venv/
+venv/
```

Comment:
.venv/

Reply:
When I try to use ".venv/" the venv folder doesn't get ignored. It works when I do "venv/". https://github.com/github/gitignore/blob/main/Python.gitignore This is shown here as well. Not sure why this is or I don't know if it's just on my machine it's like this. But happy to update it, if this is the way to go =)

Reply:
Hmm. You're quite right that in a generic sense it does have it venv in it as well as .venv, I've spoken to Adam briefly about it and we just wanted to check that your virtual environment folder is being managed by poetry because when we have poetry managing the virtual environment it uses a .venv folder. If your poetry is managing the virtual environment then all is fine and you can leave it as venv but if not I'd suggest getting poetry set up to manage the virtual environment and then double check what it has called the folder.

P.S. Looked at the other bits and agree all resolved on them.

Reply:
I just played around with Poetry now and it works as it should now. So you were right. It does use .venv folder when you configure it to create the virtualenv inside the project’s root directory. Changed it to .venv/ Thanks =) https://python-poetry.org/docs/configuration/#:~:text=If%20not%20set%20explicitly%2C%20poetry,root%20directory%20of%20the%20project. This was helpful.

Reply:
Great :)